operations/kubernetes/Kyverno.md

```diff
@@ -859,3 +859,6 @@ which takes advantage of Kyverno's `podSecurity` `subrule`, which in turn paves
 Starting from CSM 1.7, `container image verification policy` is being `Enforced`. During the upgrade to CSM 1.7, if any unsigned images need a restart, they will be blocked.
 
   **Work-around:** Customize the `check-image` policy. Change `validationFailureAction` from `Enforce` to `Audit`; then switch the policy back to `Enforce` when the upgrade is done. For more information on changing the customizations, look at [Policy customization](#policy-customization)
+* **Incorrectly copying container images from Artifactory to Nexus can block the pods**  
+When container images are copied from Artifactory to Nexus, their signatures must be copied along with them. Otherwise, the pods that use those images can be blocked.
```

Comment:
```suggestion
When container images are copied to Nexus, their signatures must be copied along with them. Otherwise, the pods that use those images can be blocked.
```